doc/src/learn/index.md

```diff
@@ -20,7 +20,7 @@ The following list includes the recent updates to Sui and the documentation:
 * Interact with the Sui network using our new [Rust SDK](../build/rust-sdk.md), a collection of Rust language [JSON-RPC wrapper and crypto utilities](https://github.com/MystenLabs/sui/tree/main/crates/sui-sdk).
 * Sui now supports development using [Microsoft Windows 11, macOS, and Linux](../build/install.md#supported-oses). See [install Sui](../build/install.md#prerequisites) for the prerequisites of each operating system.
 * This site is now available in two versions in the menu at top left: the default and stable [Devnet](https://docs.sui.io/devnet/learn) branch and the [Latest build](https://docs.sui.io/learn) upstream `main` branch. Use the `devnet` version for app development on top of Sui. Use the Latest build `main` branch for  [contributing to the Sui blockchain](../contribute/index.md) itself. Always check and submit fixes to the `main` branch.
-* `sui::id` is now `sui::object` and `VersionedID` is now `Info`. Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move.](https://github.com/MystenLabs/sui/pull/3241)
+* `sui::id` is now `sui::object` and `Info` is now `UID`. Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move](https://github.com/MystenLabs/sui/pull/3624).
```

Comment:
```suggestion
* `sui::id` is now `sui::object` and `Info` is now `UID` (as of [#3624](https://github.com/MystenLabs/sui/pull/3624)). Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move](https://github.com/MystenLabs/sui/pull/3241).
```

Reply:
Thoughts on this? The existing change is a bit weird as the second PR did not rename ID to object